src/client/Abfahrten/Components/FavList.tsx

```diff
@@ -78,11 +78,20 @@ const FavList = ({ staticContext }: Props) => {
           <MostUsed />
         </>
       ) : sortedFavs.length ? (
-        sortedFavs
+        <>
+          <FavEntryDisplay className="nonClickable" text="Favoriten" />
+          {sortedFavs}
+          <FavEntryDisplay className="nonClickable" text="Oft Gesucht" />
+          <MostUsed />
+        </>
       ) : (
         <>
-          <FavEntryDisplay data-testid="noFav" text="Keine Favoriten" />
-          <FavEntryDisplay text="Oft gesucht:" />
+          <FavEntryDisplay
+            className="nonClickable"
```

Comment:
Keine Statischen ClassNames.
```suggestion
            className={classes.nonClickable}
```